client/modules/IDE/actions/projects.js

```diff
@@ -54,3 +55,7 @@ export function deleteProject(id) {
       });
   };
 }
+
+export function sortProjects(order, orderBy) {
```

Comment:
you don't need to import the store to dispatch this action. you should use `bindActionCreators` and `mapDispatchToProps` as part of `redux` and `react-redux`